docs/reference-manual/common-practices/looping.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: "Looping"
+
+parent: Common practices
+grand_parent: Reference manual
+has_children: false
+
+nav_order: 1
+
+permalink: /docs/reference-manual/common-practices/looping.html
+category: "site-score"
+
+description: "Looping"
+
+tag: "Presentation"
+visible: true
+---
+
+# Looping
+
+## Process Loops
+
+Time based processes like automations, media and sub-scenarios can be looped for as long as their parent interval is active. When selecting a process, eiher form the object window or by clicking on the it's upper margin, the loop option will appear in the `Inspector`.
```

Comment:
Time-based processes such as automations, media and sub-scenarios can be looped for as long as their parent interval is active. When selecting a process, either from the object panel **(TODO mettre lien vers page correspondante de la doc)** or by clicking on its upper margin **(est-ce qu'on aurait pas un meilleur mot?)**, the loop option will appear in the `Inspector`

Reply:
upper margin (est-ce qu'on aurait pas un meilleur mot?)
"input bar" ? "process header" ? "process thumbnail"

Reply:
(TODO mettre lien vers page correspondante de la doc)
Maybe this section could be fleshed out and isolated as it's own section in the "Quick start" or in the "Reference manual" "Process" page ?

Reply:
- process header
- Maybe this section could be fleshed out and isolated as it's own section in the "Quick start" or in the "Reference manual" "Process" page ? => yes :) but in a further work, let's merge as-is